### PR TITLE
GHXM-6848: [GHX DXC][Logic+Integration] Easy Access Captcha - Cellphone & Tablet

### PR DESCRIPTION
**Pull request by mosunasyngenta**
Requirement
-----------------
Remove modal from library.

Solution
-----------------
- Added property renderWithoutModal to match design for captcha component.

Link: https://digitial-product-engineering.atlassian.net/browse/GHXM-6848

### CODE REVIEW

File: src/Recaptcha.js [hunk 1]
```diff
@@ -1,227 +1,260 @@
...
+      siteKey,
+      baseUrl,
+      lang,
+      style,
+      enterprise,
+      recaptchaDomain,
+      gstaticDomain,
+      hideBadge,
+      action,
+      renderWithoutModal,
```
Comment (mosunasyngenta) [inline src/Recaptcha.js:80]: This is the new property added

File: src/Recaptcha.js [hunk 2]
```diff
@@ -1,227 +1,260 @@
...
+          ref={$webView}
+        />
+        {footerComponent}
+        {renderLoading()}
+      </View>
     );
-});
+
+    return renderWithoutModal ? (
+      <View>{renderCaptchaComponent()}</View>
```
Comment (mosunasyngenta) [inline src/Recaptcha.js:241]: If renderWithoutModal is true, then renders with a View instead of a Modal component.

**Review by SantiagoSiordia — APPROVED** [review]

**Review by cesar-medina-25 — APPROVED** [review]

**Review by SantiagoSiordia — APPROVED** [review]

**Review by francis-tec — APPROVED** [review]

**Review by SantiagoSiordia — APPROVED** [review]

**Review by cesar-medina-25 — APPROVED** [review]

**Review by SantiagoSiordia — APPROVED** [review]

**Comment by SantiagoSiordia** [comment]
I don't know what's going on, I approve these changes but don't seem to be yet approved by me for some reason

**Review by tomasperezsyngenta — APPROVED** [review]
LGTM!

**Review by SantiagoSiordia — APPROVED** [review]

**Review by tomasperezsyngenta — APPROVED** [review]

**Pull request merged**